Jenkinsfile

```diff
@@ -1237,7 +956,75 @@ pipeline {
                 }
             }
             parallel {
+                stage('Coverity on CentOS 7') {
+                    // Eventually this will only run on Master builds.
+                    // Unfortunately for now, a PR build could break
+                    // the quickbuid, which would not be detected until
+                    // the master build fails.
+//                    when {
+//                        beforeAgent true
+//                        anyOf {
+//                            branch 'master'
+//                            not {
+//                                // expression returns false on grep match
+//                                expression {
+//                                    sh script: 'git show -s --format=%B |' +
+//                                               ' grep "^Coverity-test: true"',
+//                                    returnStatus: true
+//                                }
+//                            }
+//                        }
+//                    }
```

Comment:
It states why in the comment above what you selected.
We have to run this for every PR to make sure that it will work when it gets committed into master.
Down the road we need a solution that does not require quick-build in the dockerfile.

Reply:
Ahh.  I missed the comment.  Apologies.